src/libraries/System.Private.CoreLib/src/System/PasteArguments.Unix.cs

```diff
@@ -14,10 +14,10 @@ internal static partial class PasteArguments
         /// </summary>
         internal static string Paste(IEnumerable<string> arguments, bool pasteFirstArgumentUsingArgV0Rules)
         {
-            var stringBuilder = new StringBuilder();
+            var stringBuilder = new ValueStringBuilder(stackalloc char[512]);
```

Comment:
Is there a reason why we're allocating 512 instead of 256 on windows?

Reply:
Nope, I'd started with 512 and then backed off to 256 when I saw us already using 256 in the same project elsewhere, but neglected to revert this one.